container/container.scss

```diff
@@ -0,0 +1,71 @@
+.container {
+  /* width: 590px;
+  margin: 0 auto;*/
+}
+
+
+
+.header__container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  /*justify-content: space-around;*/
```

Comment:
why this rule is commented? If you do not need it - just remove. Please do this to every commented rule in your code